sql/test_db.sh

```diff
@@ -1,114 +1,227 @@
 #!/bin/bash
-# Usage: ./test_db.sh <createonly> <server> <dbname> <user> <pass>
-# Arguments other than the first are optional
-# Set <createonly> to 1 if you do not need to test the build
-# Set <createonly> to 0 if you do need to test the build
 
-set -e
+cd "$(dirname "$0")"
+
+function usage {
+    cat <<EEOOFF
+Usage: $0 [options]
+
+One of the following three options is required:
+  --ci              : Performs a CI build. This will pull credentials from the environment.
+  --create          : Interactively create a database
```

Comment:
Should `/docker/database.sh` be updated to use the new parameters? I know you have the legacy mode in the script, but I think the docker entrypoint is the only usage of the legacy mode.

On the other hand, if you don't want to make the change, lmk and I can approve this PR anyway and then I'll make the change.

Reply:
Yes, probably.

I left the legacy mode in the shell script because I *knew* there would be somewhere we didn't update, or someone would forget.

Again, happy for this to be a Later(tm) fix; it shouldn't have an impact on anything outside of dev environments.